README.md

```diff
@@ -2,7 +2,10 @@
 
 The OCI Image Format project creates and maintains the software shipping container image format spec (OCI Image Format). The goal of this specification is to enable the creation of interoperable tools for building, transporting, and preparing a container image to run.
 
-This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a [filesystem serialization](serialization.md), and an [image configuration](serialization.md#image-json-description). At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more filesystem serialization archives that will be unpacked to make up the final runnable filesystem. The image configuration includes information such as application arguments, environments, etc. The combination of the image manifest, image configuration, and one or more filesystem serializations is called the "OCI Image".
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a [filesystem serialization](layer.md), and an [image configuration](serialization.md).
```

Comment:
On Thu, Sep 08, 2016 at 10:22:08PM -0700, Brandon Philips wrote:
> > -This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a [filesystem serialization](serialization.md), and an [image configuration](serialization.md#image-json-description). At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more filesystem serialization archives that will be unpacked to make up the final runnable filesystem. The image configuration includes information such as application arguments, environments, etc. The combination of the image manifest, image configuration, and one or more filesystem serializations is called the "OCI Image".
> > +This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a [filesystem serialization](layer.md), and an [image configuration](serialization.md).
> 
> layer filesystem changeset instead of filesystem serialization?

“Filesystem serialization” is the old phrasing, and the fact that that
serialization uses changeset layers is not important, you just need
*some* way to serialize the filesystem.  Folks who are interested can
click on the link and see “ah, they're using layered changesets”, but
I don't think you need to understand that implementation detail in
your first 60 seconds of reading image-spec ;).

On the other hand, it's just a few words, so I'll switch them if the
above is not convincing ;).